.github/workflows/draft-release.yml

```diff
@@ -13,15 +13,15 @@ jobs:
           mkdir -p deps/bin
           curl -s -L -o deps/bin/jq https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64
           chmod +x deps/bin/jq
-          echo "::add-path::${PWD}/deps/bin"
+          echo "${PWD}/deps/bin" >> $GITHUB_PATH
       - name: Derive lifecycle version from branch name
         run: |
           [[ $GITHUB_REF =~ ^refs\/heads\/release/(.*)$ ]] && version=${BASH_REMATCH[1]}
           if [[ -z "${version}" ]]; then
             echo "lifecycle version not detected."
             exit 1
           fi
-          echo "::set-env name=LIFECYCLE_VERSION::$version"
+          echo "name=LIFECYCLE_VERSION::$version" >> $GITHUB_ENV
```

Comment:
The way I read the [documentation](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable), I would have expected these replacements for `set-env` to look like
```
echo "LIFECYCLE_VERSION=$version" >> $GITHUB_ENV
```
am I missing something?

Reply:
Ohhh I think I must have been very careless. Taking a look.